docs/tutorials/google/floquet_calibration_example.ipynb

```diff
@@ -140,7 +140,7 @@
     "\n",
     "from cirq_google.engine.qcs_notebook import get_qcs_objects_for_notebook\n",
     "device_sampler = get_qcs_objects_for_notebook(project_id, processor_id)\n",
-    "line_length = 35 if device_sampler.is_sampler else 20"
+    "line_length = 35 if device_sampler.is_simulator else 20"
```

Comment:
I think this logic needs to be updated, right? line_length should be 20 for simulators

Reply:
Yeah, sorry, fixed.